packages/gatsby/src/internal-plugins/functions/gatsby-node.ts

```diff
@@ -0,0 +1,184 @@
+import fs from "fs-extra"
+import glob from "globby"
+import path from "path"
+import webpack from "webpack"
+import multer from "multer"
+import * as express from "express"
+
+import { urlResolve } from "gatsby-core-utils"
+
+import { internalActions } from "../../redux/actions"
+
+import { ParentSpanPluginArgs, CreateDevServerArgs } from "gatsby"
+
+import TerserPlugin from "terser-webpack-plugin"
```

Comment:
```suggestion
import { urlResolve } from "gatsby-core-utils"
import { ParentSpanPluginArgs, CreateDevServerArgs } from "gatsby"
import TerserPlugin from "terser-webpack-plugin"
import { internalActions } from "../../redux/actions"
```

code style (import order + spaces we use in other places)